ElectricEquipment1/Controllers/UserController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using ElectronicEquipment.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data.SqlClient;
+using System.Data;
+using Microsoft.AspNetCore.Cors;
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+
+namespace ElectronicEquipment.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    [EnableCors("AllowOrigin")]
+    public class UserController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        public readonly UserContext _context;
+        public UserController(IConfiguration configuration,UserContext context)
+        {
+            _configuration = configuration;
+            _context = context;
+        }
+        [AllowAnonymous]
```

Comment:
Why we allowing anonymous?

Reply:
To avoid the authentication for adding and login the user